model/userDao.js

```diff
@@ -0,0 +1,64 @@
+const { dataSource } = require('./dataSource');
+
+const createUser = async (
+  email,
+  password,
+  name,
+  phoneNumber,
+  address,
+  gender,
+  birthDate,
+  points
+) => {
+  try {
+    return await dataSource.query(
+      `
+        INSERT INTO users (
+            email,
+            password,
+            name,
+            phone_number,
+            address,
+            gender,
+            birth_date,
+            points
+        ) VALUES (
+          ?, ?, ?, ?, ?, ?, ?, ?
+        )
+      `,
+      [email, password, name, phoneNumber, address, gender, birthDate, points]
+    );
+  } catch (err) {
+    console.log(err);
+    err = new Error(err.message);
+    err.statusCode = 500;
```

Comment:
상태코드 500은 처리하지 못한 에러에 대한 코드 입니다. 따라서 400번대 코드로 바꿔주세요.
```suggestion
    err.statusCode = 400;
```